Makefile

```diff
@@ -40,8 +41,12 @@ manzan-installer-v%.jar: /QOpenSys/pkgs/bin/zip appinstall.jar
 	system "dltlib ${BUILDLIB}" || echo "could not delete"
 	system "crtlib ${BUILDLIB}"
 	system "dltlib ${BUILDLIB}"
+	> config/app.ini > config/data.ini > config/dests.ini
 	rm -fr /QOpenSys/etc/manzan
+	rm -fr /opt/manzan
 	gmake -C config BUILDVERSION="$*" install BUILDLIB=${BUILDLIB}
 	gmake -C ile BUILDVERSION="$*" BUILDLIB=${BUILDLIB}
 	gmake -C camel BUILDVERSION="$*" clean install
-	/QOpenSys/QIBM/ProdData/JavaVM/jdk80/64bit/bin/java -jar appinstall.jar --qsys manzan --dir /QOpenSys/etc/manzan --file /opt/manzan -o $@
+	install -m 555 -o qsys service-commander-def.yaml ${INSTALL_ROOT}/opt/manzan/lib/manzan.yaml
+	ln -sf ${INSTALL_ROOT}/opt/manzan/lib/manzan.yaml /QOpenSys/etc/sc/services/manzan.yaml
+	/QOpenSys/QIBM/ProdData/JavaVM/jdk80/64bit/bin/java -jar appinstall.jar --qsys manzan --file /QOpenSys/etc/manzan --file /opt/manzan --file /QOpenSys/etc/sc/services/manzan.yaml -o $@

```

Comment:
@ThePrez Looks like the app installer resolves symbolic links which prevents the installer from bundling in and being able to install `/QOpenSys/etc/sc/services/manzan.yaml`. Any idea how to get around this?

![image](https://github.com/user-attachments/assets/289e8ab8-99bc-45d9-a400-93e07f0d75b5)